From 4f8c7a12f84841b8722e4a72c4c5ec66cf98ac78 Mon Sep 17 00:00:00 2001
From: Jonathon Hall <jonathon.hall@privateinternetaccess.com>
Date: Thu, 27 May 2021 14:46:27 -0400
Subject: [PATCH 2/3] Remove win/walk deps (not needed for e-d-s), use local
 wireguard-go

---
 go.mod | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/go.mod b/go.mod
index 7a513156..75b2b5c4 100644
--- a/go.mod
+++ b/go.mod
@@ -3,8 +3,6 @@ module golang.zx2c4.com/wireguard/windows
 go 1.16
 
 require (
-	github.com/lxn/walk v0.0.0-20210112085537-c389da54e794
-	github.com/lxn/win v0.0.0-20210218163916-a377121e959e
 	golang.org/x/crypto v0.0.0-20210322153248-0c34fe9e7dc2
 	golang.org/x/net v0.0.0-20210316092652-d523dce5a7f4
 	golang.org/x/sys v0.0.0-20210320140829-1e4c9ba3b0c4
@@ -13,6 +11,5 @@ require (
 )
 
 replace (
-	github.com/lxn/walk => golang.zx2c4.com/wireguard/windows v0.0.0-20210121140954-e7fc19d483bd
-	github.com/lxn/win => golang.zx2c4.com/wireguard/windows v0.0.0-20210224134948-620c54ef6199
+	golang.zx2c4.com/wireguard => ..\wireguard-go
 )
-- 
2.31.1.windows.1

