From 1d3a44e1eb420b7e73c492a132cef488e6b888f6 Mon Sep 17 00:00:00 2001
From: Jonathon Hall <jhall@londontrustmedia.com>
Date: Wed, 6 Mar 2019 00:41:57 +0900
Subject: [PATCH 1/8] Patch msvc-build.bat and openvpnserv.vcxproj

---
 msvc-build.bat | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/msvc-build.bat b/msvc-build.bat
index 8256c62d..b7cb0505 100644
--- a/msvc-build.bat
+++ b/msvc-build.bat
@@ -7,8 +7,10 @@ setlocal ENABLEDELAYEDEXPANSION
 cd /d %0\..
 call msvc-env.bat
 
-set PLATFORMS=x64
-set CONFIGURATIONS=Debug Release
+if "%PLATFORMS%"=="" set PLATFORMS=x64
+if "%CONFIGURATIONS%"=="" set CONFIGURATIONS=Release
+if "%TOOLSET%"=="" set TOOLSET=v142
+if "%SDKVERSION%"=="" set SDKVERSION=10.0
 
 if exist "%VCHOME%\vcvarsall.bat" (
 	call "%VCHOME%\vcvarsall.bat"
@@ -32,7 +34,7 @@ for %%p in (%PLATFORMS%) do (
 			vcbuild /errfile:error.log /showenv "%SOLUTION%" /rebuild /platform:%%p "%%c|%%p"
 			for %%f in (error.log) do if %%~zf GTR 0 goto error
 		) else  (
-			msbuild "%SOLUTION%" /p:Configuration="%%c" /p:Platform="%%p"
+			msbuild "%SOLUTION%" /p:Configuration="%%c" /p:Platform="%%p" /p:PlatformToolset="%TOOLSET%" /p:TargetPlatformVersion="%SDKVERSION%"
 			if errorlevel 1 goto error
 		)
 	)
-- 
2.20.1

