From 93809072bca576694a7f327a75c1df4f497b0d04 Mon Sep 17 00:00:00 2001
From: Jonathon Hall <jhall@londontrustmedia.com>
Date: Fri, 22 Mar 2019 09:21:21 -0400
Subject: [PATCH 7/8] Enable AEAD ciphers (AES-*-GCM) on Windows

---
 config-msvc.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/config-msvc.h b/config-msvc.h
index 312f84bd..9dcf1195 100644
--- a/config-msvc.h
+++ b/config-msvc.h
@@ -5,6 +5,7 @@
 #define ENABLE_DEF_AUTH 1
 #define ENABLE_PF 1
 #define ENABLE_CRYPTO_OPENSSL 1
+#define HAVE_AEAD_CIPHER_MODES 1
 #define ENABLE_DEBUG 1
 #define ENABLE_EUREPHIA 1
 #define ENABLE_FRAGMENT 1
-- 
2.30.0.windows.2

