From 427d4d44956f874964cc1e3a4e739794a4abfddf Mon Sep 17 00:00:00 2001
From: Jonathon Hall <jhall@londontrustmedia.com>
Date: Wed, 15 Jan 2020 21:39:28 -0500
Subject: [PATCH 8/8] Enable EC crypto on Windows

---
 config-msvc.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/config-msvc.h b/config-msvc.h
index 9dcf1195..16f8cea9 100644
--- a/config-msvc.h
+++ b/config-msvc.h
@@ -112,7 +112,6 @@
 #define HAVE_RSA_METH_SET0_APP_DATA 1
 #define HAVE_RSA_METH_GET0_APP_DATA 1
 #define HAVE_EC_GROUP_ORDER_BITS 1
-#define OPENSSL_NO_EC 1
 #define HAVE_EVP_CIPHER_CTX_RESET 1
 #define HAVE_DIINSTALLDEVICE 1
 
-- 
2.30.0.windows.2

