From c96ffe388a8fdab66c87da5d46562c85e4288233 Mon Sep 17 00:00:00 2001
From: Jonathon Hall <jhall@londontrustmedia.com>
Date: Fri, 8 Mar 2019 03:02:48 +0900
Subject: [PATCH 4/8] Disable non-blocking connect

---
 src/openvpn/syshead.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/openvpn/syshead.h b/src/openvpn/syshead.h
index 8342eae0..d07e01f8 100644
--- a/src/openvpn/syshead.h
+++ b/src/openvpn/syshead.h
@@ -628,9 +628,13 @@ socket_defined(const socket_descriptor_t sd)
 /*
  * Is non-blocking connect() supported?
  */
+/* Non-blocking connect has a rubbish poll loop; blocking connect is much faster in practice */
+#undef CONNECT_NONBLOCK
+/*
 #if defined(HAVE_GETSOCKOPT) && defined(SOL_SOCKET) && defined(SO_ERROR) && defined(EINPROGRESS) && defined(ETIMEDOUT)
 #define CONNECT_NONBLOCK
 #endif
+*/
 
 /*
  * Compression support
-- 
2.20.1

