From 47ef12dcbefc2d470e230bf272ceb04c08246431 Mon Sep 17 00:00:00 2001
From: Jonathon Hall <jonathon@privateinternetaccess.com>
Date: Fri, 15 Jan 2021 14:19:46 -0500
Subject: [PATCH 3/3] Fix build command in build.bat

---
 msvc/build.bat | 17 +++++++++++++----
 1 file changed, 13 insertions(+), 4 deletions(-)

diff --git a/msvc/build.bat b/msvc/build.bat
index 9a71868..f5bfb70 100644
--- a/msvc/build.bat
+++ b/msvc/build.bat
@@ -188,10 +188,19 @@ cd build.tmp\openvpn*
 xcopy * ..\..\..\..\%OPENVPN_BUILD_OPENVPN% /E
 cd ..\..\..\..\%OPENVPN_BUILD_OPENVPN%
 
-if %ARCH%==32 ( SET PLAT=Win32 ) else ( SET PLAT=x64 )
-msbuild openvpn.sln /p:Platform=%PLAT% /p:Configuration=%RELEASE%
-if errorlevel 1 goto error
-copy x64-Output\%RELEASE%\*.exe "%TARGET%\bin"
+if %ARCH%==32 (SET PLAT=Win32) else (SET PLAT=x64)
+rem The upstream script calls msbuild directly here, but it needs environment
+rem variables that are set up by msvc-build.bat and msvc-env.bat [notably
+rem SOURCEBASE], call msvc-build.bat instead.  [note that this is also patch by
+rem a patch on OpenVPN to allow us to set the platform, configuration, etc.]
+rem msbuild openvpn.sln /p:Platform=%PLAT% /p:Configuration=%RELEASE%
+set OPENVPN_DEPROOT=%TARGET%
+set PLATFORMS=%PLAT%
+set CONFIGURATIONS=%RELEASE%
+call msvc-build.bat
+if errorlevel 1 goto error
+echo "copying %PLAT%-Output\%RELEASE%\*.exe"
+copy %PLAT%-Output\%RELEASE%\*.exe "%TARGET%\bin"
 if errorlevel 1 goto error
 copy include\openvpn-*.h "%TARGET%\include"
 if errorlevel 1 goto error
-- 
2.20.1

